config.toml

```diff
@@ -40,6 +40,18 @@
     # Make sure that this is greater than the unbonding period!
     SetGuardianEpochsDelay = 20 # 2 times the normal unstaking period
 
+# ChainParametersByEpoch defines chain operation configurable values that can be modified based on epochs
+ChainParametersByEpoch = [
+    { EnableEpoch = 0, RoundDuration = 6000, ShardConsensusGroupSize = 63, ShardMinNumNodes = 400, MetachainConsensusGroupSize = 400, MetachainMinNumNodes = 400, Hysteresis = 0.2, Adaptivity = false },
```

Comment:
The parameters that were removed from _nodeSetup_ (moved here) :heavy_check_mark: